testing-pre-releases.md

```diff
@@ -0,0 +1,264 @@
+
+Pre-compiled CI/CD releases of .deb or .rpm packages are deployed into the `gs://kubernetes-release-dev/bazel/`
+GCS bucket.
+
+To explore versions available in Google Storage buckets use:
+
+```bash
+gsutil ls gs://kubernetes-release-dev/bazel/{filter}
+
+# e.g. search all CI/CD v1.10 releases
+gsutil ls -g gs://kubernetes-release-dev/bazel/v1.10*
+```
+
+As alternative, you can browse using <https://console.cloud.google.com/storage/browser/kubernetes-release-dev/bazel/> .
+
+To retrieve a pre-compiled  CI/CD releases of .deb or .rpm version of kubeadm binary use:
+
+```bash
+gsutil cp {bucket-name}/{release}/bin/linux/amd64 *.deb .
+```
+
+### Getting kubeadm binaries from a GCS bucket
+
+Pre-compiled GA, alpha/beta versions of kubeadm binary are deployed into `gs://kubernetes-release/release/` GCS bucket,
+while CI/CD versions are deployed into `gs://kubernetes-release-dev/ci-cross/` bucket.
+
+To explore versions available in Google Storage buckets use:
+
+```bash
+gsutil ls {bucket-name}{filter}
+
+# e.g. search all GA and alpha/beta v1.10 releases
+gsutil ls -g gs://kubernetes-release/release/v1.10*
+```
+
+As alternative, you can browse GCS buckets using <https://console.cloud.google.com/storage/browser/{bucket-name}> or
+<http://gcsweb.k8s.io/gcs/kubernetes-release/release/> (only for `gs://kubernetes-release/release/`).
+
+To retrieve a pre-compiled version of kubeadm binary use:
+
+```bash
+gsutil cp {bucket-name}/{release}/bin/linux/arm64/kubeadm .
+
+# or, only for releases in gs://kubernetes-release/release/
+curl -L https://dl.k8s.io/release/{release}/bin/linux/amd64/kubeadm && chown +x kubeadm
+```
+
+### Getting docker images from a GCR registry
+
+Kubeadm will take care of downloading images corresponding to the `--kubernetes-version` flag of kubeadm or the `kubernetesVersion` field of Kubeadm API.
+
+For GA, alpha/beta valid version numbers are:
+- Semantic version number
+- Kubernetes release labels for GA, alpha/beta versions like e.g. `stable`, `stable-1`, `stable-1.10`, `latest`,
+  `latest-1`, `latest-1.10`. See <https://console.cloud.google.com/storage/browser/kubernetes-release/release/>
+  for the full list of labels.
+
+For CI/CD valid version numbers are:
+- prefix `ci/` followed by a semantic version number
+- prefix `ci/` followed by Kubernetes release labels for CI/CD versions like e.g. `ci/latest`, `ci/latest-1`, `latest-1.10`.
+  See <https://console.cloud.google.com/storage/browser/kubernetes-release-dev/ci-cross/> for the full list of labels.
+
+Instead, if you want to retrieve manually pre-compiled/pre-built GA, alpha/beta versions of control plane images, such
+images are deployed into `gcr.io/google-containers` GCR registry, while CI/CD versions are deployed into `gcr.io/kubernetes-ci-images` GCR registry.
+
+To explore versions available in Google Container Registry use
+
+```bash
+gcloud container images list-tags gcr.io/{gcr-repository-name}/{image-name} --sort-by=~tags --filter=tags:v1.10.2 --limit=50
```

Comment:
it's kind of weird that some of the fields are placeholders (iimage name) but tags is already filled in. Maybe have one example with all fields empty and another with all fields full?